src/main/java/com/avispl/dal/communicator/polycom/videoos/PolycomVideoOS.java

```diff
@@ -0,0 +1,1046 @@
+        response.iterator().forEachRemaining(jsonNode -> {
+            String langtag = getJsonProperty(jsonNode, "langtag", String.class);
+            ArrayNode stateList = getJsonProperty(jsonNode, "stateList", ArrayNode.class);
+
+            if (langtag != null && stateList != null) {
+                statistics.put("System Status#" + normalizeStringPropertyName(
+                        langtag.replaceAll("_", " ")),
+                        stateList.get(0).asText().replaceAll("_", " ").toUpperCase());
+            }
+        });
+    }
+
+    /**
+     * Change property name to start with a capital letter
+     *
+     * @param string parameter name
+     * @return {@link String} property name with first capital letter
+     */
+    private String normalizeStringPropertyName(String string) {
+        return string.substring(0, 1).toUpperCase() + string.substring(1).toLowerCase();
+    }
+
+    /**
+     * Retrieve the current state of the collaboration session
+     *
+     * @return Integer value of the current device volume level
+     * @throws Exception during http communication
+     */
+    private void retrieveCollaborationStatus(Map<String, String> statistics) throws Exception {
+        JsonNode response = doGet(COLLABORATION, JsonNode.class);
+        if (!response.isNull()) {
+            String sessionState = getJsonProperty(response, "state", String.class);
+            statistics.put("Collaboration#Session State", sessionState);
+            if ("ACTIVE".equals(sessionState)) {
+                statistics.put("Collaboration#Session ID", getJsonProperty(response, "id", String.class));
+            }
+        }
+    }
+
+    /**
+     * Retrieve current volume level of the device
+     *
+     * @return Integer value of the current device volume level
+     * @throws Exception during http communication
+     */
+    private Integer retrieveVolumeLevel() throws Exception {
+        return doGet(VOLUME, Integer.class);
+    }
+
+    /**
+     * Update device's volume with a given value
+     *
+     * @param value target value for the device volume
+     * @return boolean value indicating the success of the operation
+     * @throws Exception during http communication
+     */
+    private boolean updateVolumeLevel(int value) throws Exception {
+        return doPost(VOLUME, value).getStatusCode().is2xxSuccessful();
+    }
+
+    /**
+     * Retrieve stats of currently active conference call
+     *
+     * @param statistics map to put values to
+     * @return active conference id (#0), -1 if there are no active conferences available
+     * @throws Exception during http communication
+     */
+    private Integer retrieveActiveConferenceCallStatistics(Map<String, String> statistics) throws Exception {
+        ArrayNode conferenceCalls = listConferenceCalls();
+        if (conferenceCalls.size() <= 0) {
+            return -1;
+        }
+        JsonNode activeConference = conferenceCalls.get(0);

```

Comment:
Similar to other comment above. Whey are we using the first conference in the list?

Reply:
Added to javadocs. Clarified it with Connor Gray also - even though it's no explicitly defined by the REST API reference doc - there's no normal way for creating more than 1 conference at the same time.

Reply:
I still no sure I'm comfortable with it. If it is guaranteed that there will be only one conference, why not to check for it and throw error if there are more than one reported? It is better to throw an error than use wrong conference

Reply:
Agree, throwing an exception if there's more than 1 conference at a time